python/cuvs/cuvs/neighbors/prefilters/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .prefilters import Prefilter, from_bitmap, no_filter
```

Comment:
Can we please rename this to 'filters' instead of 'prefilters'? Technically the filters don't need to change whether they are done pre- or post-query, and thus these are filters, not just prefilters.